deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -240,6 +240,16 @@ def get_protocols(all_protocols):
     return [valid_protocols, invalid_protocols]
 
 
+def callback_active_response_received():
+    msg = fr"DEBUG: Active response request received: (local_source) \[\] NRN 002 restart-wazuh0 admin 1.1.1.1 1.1 44 (agente-cualquiera) any->/carpeta/testing - -"
+    return monitoring.make_callback(pattern=msg, prefix=monitoring.REMOTED_DETECTOR_PREFIX)
+
+
+def callback_active_response_sent():
+    msg = fr"DEBUG: Active response sent: #!-execd restart-wazuh0 admin 1.1.1.1 1.1 44 (agente-cualquiera) any->/carpeta/testing - -"
```

Comment:
"agente-cualquiera"

Reply:
"carpeta/testing"

Please, be careful with this, the project language is English!

Reply:
Done 20cadf88fd2d3f4777f24434f6ca2b8e95ff0f11